aten/src/THC/THCReduceAll.cuh

```diff
@@ -31,9 +31,7 @@ template <typename T,
           typename ReduceOp,
           int ADims>
 __global__ void
-#if defined(__HIP_PLATFORM_HCC__)
```

Comment:
from my cursory look this whole file is deadcode (but that's outside of this PR scope)

Reply:
Ha!